sunpy/net/dataretriever/sources/stereo.py

```diff
@@ -140,24 +146,23 @@ def _get_url_for_timerange(self, timerange, **kwargs):
             raise ValueError('Earliest date for which PLASTIC data is available is {:%Y-%m-%d}'.format(START_DATE))
         prefix = 'http://stereo-ssc.nascom.nasa.gov/data/beacon/{source}/plastic/'
         prefix_copy = prefix
-        if (source == 'ahead'):
+        if (source == 'STEREO_A'):
             prefix += '%Y/%m/ST{char}_LB_PLASTIC_%Y%m%d_'
             prefix_copy += '%Y/%m/ST{char}_LB_PLA_BROWSE_%Y%m%d_'
-        else:
+        elif (source == 'STEREO_B'):
             prefix += '%y/%m/ST{char}_LB_PLASTIC_%Y%m%d_'
             prefix_copy +='%y/%m/ST{char}_LB_PLA_BROWSE_%Y%m%d_'
```

Comment:
what's the difference between the two above? just the first `%Y`/`%y`? then that should be the only thing between this `if` block... which could be converted into a dictionary.